modules/gdnative/godot.cpp

```diff
@@ -35,6 +35,7 @@
 #include "global_config.h"
 #include "global_constants.h"
 #include "variant.h"
+#include "os/os.h"
```

Comment:
Yeah, sorry, I was not aware about clang-format when creating the PRs. Will update them soon. Perhaps it makes sense to mention it contribution guide?